docs/content/articles/pt/madara-beast-article.md

```diff
@@ -0,0 +1,157 @@
+   Independente da crate escolhida, a estrutura subjacente utiliza a Cairo VM. A
+   linguagem Cairo facilita a criação de programas prováveis, permitindo a
+   demonstração da execução correta do cálculo.
+2. **Settlement:** Como um settlement rollup, o estado da appchain do Madara
+   pode ser reconstruído exclusivamente ao examinar a sua settlement layer. Ao
+   se estabelecer mais frequentemente na Starknet L2, uma appchain L3 consegue
+   alcançar uma finalidade dura mais rápido, enquanto a descentralização da
+   camada de sequenciamento permite uma finalidade mais robusta e suave. Assim,
+   a settlement é reforçada em ambas as frentes (finalidade dura e suave).
+3. **Sequenciamento:** O Madara assume o controlo do processo de sequenciação,
+   que pode ser alterado em função das necessidades da aplicação - seja o
+   simples FCFS, PGA ou esquemas mais complexos como Narwhall & Bullshark.
+   Certas appchains podem escolher implementar mempools encriptados para
+   garantir uma ordenação justa e mitigar o impacto do MEV.
+4. **Disponibilidade de Dados:** A Disponibilidade de dados garante o acesso à
+   árvore de estados completa, oferecendo aos utilizadores a confiança de que
+   podem provar a posse dos seus fundos mesmo se o Madara sofrer uma
+   interrupção. Madara irá oferecer aos desenvolvedores uma data de esquemas de
+   disponibilidade de dados para escolherem.
+5. **Controlo:** Cada appchain do Madara pode escolher o seu modelo de controlo.
+   [Snapshot X](https://twitter.com/SnapshotLabs) oferece um sistema de controlo
+   totalmente on-chain que depende de provas de armazenamento. Mecanismos de
+   controlo alternativos também estão sob exploração, como o native substrate
+   governance pallet. Controlo on-chain é um valor fundamental para o Madara.
+
+![come come](https://lh4.googleusercontent.com/i7bXi2IPV-LTLzEgueA2SPHGULUFDj1OX4IznOQr5BeZe0hcey-VXA5TOV6q9XaVqBGAcYiie7u7uxw7q1ByZxjkPQKHERqKJTxhdDdTSgBQy8smyNO3jEHiNJv7Eqh8BMxjj4fFlQAW6gm-hQMzyIU)
+
+## Enter: Madara
+
+No Madara, a Cairo VM está a ser aprimorada utilizando a framework Substrate e
+integrando a Cairo VM para executar programas Cairo e smart contracts da
+Starknet. Substrate é uma framework do Rust open-source para construir
+blockchains personalizáveis, que é conhecida pela sua flexibilidade. Enquanto
+isso, a Cairo VM é desenhada especificamente para gerar Provas de Validade
+eficientemente para execução de programas. Ao implementar rastreamento de estado
+e um smart contract para verificar essas provas na L2, a appchain garante uma
+integração segura com a Starknet. Desta maneira, o Madara aproveita o poder do
+Cairo para permitir a comprovação da execução do programa.
+
+A natureza modular inerente da framework Substrate permite ao desenvolvedores
+personalizar a appchain com facilidade. Nenhuma suposição é imposta, permitindo
+que você incorpore seus protocolo de consenso próprio, funções hash, esquema de
```

Comment:
permitindo a incorporação dos teus protocolos de consenso, funções ... 
u started informal and finished formal?